jwql/website/apps/jwql/archive_database_update.py

```diff
@@ -266,11 +267,11 @@ def files_in_filesystem(files, permission_type):
             full_filepath = os.path.join(FILESYSTEM, permission_type, relative_filepath)
             filenames.append(full_filepath)
         except ValueError:
-            print('Unable to determine filepath for {}'.format(filename))
+            logging.warning('Unable to determine filepath for {}'.format(filename))
     return filenames
 
 
-def update_database_table(update, instrument, prop, obs, thumbnail, obsfiles, types, startdate, enddate, proposal_category):
+def update_database_table(update, instrument, prop, obs, thumbnail, obsfiles, types, startdate, enddate, proposal_category, obsfilenames=[]):
```

Comment:
Please update the docstring here as well 😎

Reply:
Done.